packages/babel-compat-data/data/plugin-bugfixes.json

```diff
@@ -1,125 +1,125 @@
 {
   "transform-async-to-generator": {
     "chrome": "55",
+    "opera": "42",
```

Comment:
The change of support matrix is related to object key orders only.

Reply:
dang that's annoying haha